charts/connect/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "redpanda-connect.fullname" . }}-test-connection"
+  labels:
+    {{- include "redpanda-connect.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+    "kube-linter.io/ignore-all": "Linter does not need to run on this test"
```

Comment:
Should kube-linter be running at somepoint? I don't believe we have it configured in this repo but it would be good to.

Reply:
Copied this over from the stand alone repo. I'm not sure why this annotation is here, kube-lint wasn't running in CI there either.

I ran `kube-lint` on the repo just now and there are 52 errors, I'd rather not make fixing all of those a blocker for this PR. I think it would be best to add kube-lint in a follow-up PR.